index.html

```diff
@@ -905,200 +297,201 @@ <h1><a name="2NormRefs">2 Normative references</a></h1>
 Coding-independent code points for video signal type identification<br class="xhtml" />
  <code><a href=
 "https://www.itu.int/rec/R-REC-BT.709">https://www.itu.int/rec/R-REC-BT.709</a></code></p>
-
+</section>
 
 <!-- ************Page Break******************* -->
 <!-- ************Page Break******************* -->
-<h1><a name="3Defsandabbrevs">3 Terms, definitions, and
-abbreviated terms</a></h1>
+<section>
+<h2>Terms, definitions, and
+abbreviated terms</h2>
 
-<h2><a name="3Definitions">3.1 Definitions</a></h2>
```

Comment:
Note that replacing IDs like this will **break all incoming links to the PNG specification**, so is a very bad idea.

Reply:
Agreed.

I was thinking of per-version URLs like:
https://www.w3.org/TR/2003/REC-PNG-20031110/
https://www.w3.org/TR/2003/PR-PNG-20030520/

If the new spec gets a new URL, there are no links to break.

But there is a latest-version URL where links should be preserved, IE: https://www.w3.org/TR/PNG/#1Scope

I'll make sure the links are preserved.

Reply:
Latest version is what most people link to.